tests/unit/migration/test_run.py

```diff
@@ -223,21 +233,22 @@ async def test_run_migration(migration_check_persistent, user, get_credential):
 async def test_data_warehouse_migration_get_session(
     mock_session_manager,
     migration_check_persistent,
-    user,
     test_dir,
+    user,
+    app_container,
 ):
     """Test data warehouse migration get_session method"""
     fixture_path = os.path.join(test_dir, "fixtures/request_payloads/feature_store.json")
-    feature_store_user_id = ObjectId()
```

Comment:
I think setting the feature store user id to a different value than `user.id` was the intention of this test.

Reply:
Yes. That is the purpose of this test. I should add more comment on this test. 😅

Reply:
👍 For adding comment